### DISCUSSION TITLE
https://notes.alexkehayias.com/coming-back-to-rust-after-4-years/

### DISCUSSION THREAD

**Discussion by giscus** [alexkehayias/note-comments#13] [Announcements]
# https://notes.alexkehayias.com/coming-back-to-rust-after-4-years/

I recently picked up rust for a personal infrastructure project and was amazed at the amount of progress on the language and tooling over the years.

https://notes.alexkehayias.com/coming-back-to-rust-after-4-years/

**Comment by peterpeerdeman** [comment]
Hi Alex, just wanted to say hi and thank you for sharing your wonderful collection of notes 🙏 This specific rust note has helped me finally pull the trigger on learning Rust, and it has been [a great journey](https://notes.peterpeerdeman.nl/learning-rust) so far.

**Reply by alexkehayias** [reply]
Thank you Peter! Looks like you’ve dug into rust quite a bit already. 

It took me about 2 years to learn and feel fluent in rust (and a few false starts). Changed the way I think about programming.
